src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $white-color;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    color: $secondary-text-color;
+
+  }
+
+  &__rewiew-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    display: flex;
+    color: $main-text-color;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    color: $main-text-color;
+    font-size: 16px;
+    font-weight: 700;
+
+    &-label {
+      color: $secondary-text-color;
+      font-size: 12px;
+    }
+  }
+
+  &__buy {
+    display: flex;
+    height: 40px;
+    padding: 12px 69px;
```

Comment:
```suggestion
    padding: 12px 0;
    width: 100%;
```

Reply:
The text of the button can be changed it shouldn't have an impact on the size of the button on the card